### ISSUE TITLE
The classical multi-class code performs no multi-class task.

### ISSUE THREAD

**Issue by garylavayou** [vinayakumarr/Network-Intrusion-Detection#12]
The code in [all.py](https://github.com/vinayakumarr/Network-Intrusion-Detection/blob/master/NSL-KDD/classical/multiclass/all.py#L8) is almost the same as that in `../binary/all.py`, except the part of `GridSearch`.

In the code, there is no multi-class classification task.

**Comment by vinayakumarr** [comment]
Yes, both are same. The same code (all.py) can be used for both binary and multiclass classification.
